README.md

```diff
@@ -117,9 +146,9 @@ profile_helper.
 
 #### Tmux
 
-You can use this hook if you have installed [base16-tmux][3] through
-[TPM][10]. base16-shell will update (or create) the
-`$HOME/.config/base16-project/tmux.base16.conf` file and set the
+You will automatically use this hook if you have installed
+[base16-tmux][3] through [TPM][10]. base16-shell will update (or create)
+the `$HOME/.config/base16-project/tmux.base16.conf` file and set the
```

Comment:
I worry a little bit about this location, since we're still figuring out the rename. It might be better to use `$HOME/.config/base16/tmux.base16.conf` since it's still a base16 template at the end of the day.

Reply:
I’ll leave this here for now since that config path wasn’t introduced in this PR, but we can create a new PR that changes that directory name.

Reply:
That works